homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+Javascript (JS) is a scripting languages, primarily used on the Web. It is used to enhance HTML pages and is commonly found embedded in HTML code.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. Edit a commit message
+2. Remove a file from GIT without removing from the file system
+3. Find a commit that broke something after a merge
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it and make the repo to your own.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You copy it but you do not have the right to edit the parent file.
```

Comment:
I think you can edit all files , not only parent files, and the files will be stored locally in the git repository instead of being in the remote one  :-)